packages/utilities/src/array.ts

```diff
@@ -15,7 +15,9 @@ export function createArray(size: number, getItem?: (index?: number) => any) {
   let array = [];
 
   for (let i = 0; i < size; i++) {
-    array.push(getItem(i));
+    if (getItem) {
```

Comment:
How about just making getitem required? It's a silly method if you don't provide getItem